From a85de9bb74ea4b9ff7cdfb4c9b945b7fb4030cab Mon Sep 17 00:00:00 2001
From: Thor Thayer <thor.thayer@linux.intel.com>
Date: Fri, 2 Feb 2018 11:44:13 -0600
Subject: [PATCH 07/25] FogBugz #532583-3: mtd: a10sr: Add A10SR HWMON

commit  f6ea55a141f4596ee406264f7fce57155b444e0e from
https://github.com/altera-opensource/linux-socfpga.git

Add HWMON to the A10 System Resource Multi-Function Device.
This fixes a previous patch
['commit b5678c1b894d ("FogBugz #411698-2: Add Altera A10-SR
                        power supply alarms")']
that removed this change when cherry-picked.

Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mfd/altera-a10sr.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mfd/altera-a10sr.c b/drivers/mfd/altera-a10sr.c
index 96e7d2c..c57b549 100644
--- a/drivers/mfd/altera-a10sr.c
+++ b/drivers/mfd/altera-a10sr.c
@@ -30,6 +30,10 @@
 
 static const struct mfd_cell altr_a10sr_subdev_info[] = {
 	{
+		.name = "altr_a10sr_hwmon",
+		.of_compatible = "altr,a10sr-hwmon",
+	},
+	{
 		.name = "altr_a10sr_gpio",
 		.of_compatible = "altr,a10sr-gpio",
 	},
-- 
2.8.1

